### PR TITLE
docs: minor improvements to permissions documentation

### PR DESCRIPTION
**Pull request by BalduinLandolt**
### Task Description/Number

Issue Number: None

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [ ] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [x] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

**Review by mpro7 — APPROVED** [review]
Just question about `Knora`, should it be `DSP-API` instead?

**Comment by BalduinLandolt** [comment]
> Just question about `Knora`, should it be `DSP-API` instead?

you're right, I missed some instances. (except of course `knora-base` and `knora-admin` where changing it would be a bigger thing)

File: mkdocs.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ nav:
               - 0004 Change Triplestore Service Manager and Fuseki implementation to ZLayer: 05-internals/design/adr/ADR-0004-change-triplestore-service-manager-and-fuseki-implementation-to-zlayer.md
               - 0005 Change ResponderManager to a simple case class: 05-internals/design/adr/ADR-0005-change-respondermanager-to-a-simple-case-class.md
               - 0006 Gradually Replace AKKA-HTTP with ZIO-HTTP: 05-internals/design/adr/ADR-0006-use-zio-http.md
+              - 0007 ZIO-fication of Responders: 05-internals/design/adr/ADR-0007-zio-fication-of-responders.md
```
Comment (seakayone) [inline mkdocs.yml:68]: Good finding! Thanks, I seem to have missed that when adding the document.

**Review by seakayone — APPROVED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (BalduinLandolt) [reply mkdocs.yml:68]: I only found it because I ran the docs locally, and then it shows warnings for unused files :)

**Pull request merged**